.travis.yml

```diff
@@ -8,6 +8,7 @@ services:
 install:
     - sudo pip install -q sphinx --use-mirrors
     - wget -q -O conf.py https://raw.githubusercontent.com/ongr-io/docs-aggregator/master/source/conf-travis.py
+    - composer self-update
```

Comment:
Let Travis manage third party updates. Don't worry due "30 days" warning in travis logs. We won't get any benefits updating composer each time. However, it could crash build if composer could not update.